.github/workflows/django.yml

```diff
@@ -7,29 +7,27 @@ jobs:
     strategy:
       max-parallel: 4
       matrix:
-        python-version: ['3.8', '3.9', '3.10', '3.11', '3.12']
-        django-version: ['3.2.0', '4.0.0', '4.1.0', '4.2.0', '5.0.0', 'main']
+        python-version: ['3.8', '3.9', '3.10', '3.11', '3.12', '3.13']
```

Comment:
We could also drop support for Python 3.8 since it's end of life now, although fine with keeping it if it isn't hurting us yet.

Reply:
Django 4.2 still officially supports Python 3.8, which is why I've left it in the test matrix.